conf/application.conf

```diff
@@ -67,10 +67,28 @@ auth.client.id="fake_auth_id"
 auth.client.secret="fake_auth_secret"
 auth.client.id=${?AUTH_ID}
 auth.client.secret=${?AUTH_SECRET}
+
+# Google Oauth Config Example Config
+#auth.client.token_url="https://accounts.google.com/o/oauth2/token"
+#auth.client.profile_url="https://www.googleapis.com/oauth2/v2/userinfo"
+#auth.response.name_field="email"
+#auth.response.sub_field="id"
+#auth.response.email_field="email"
+#auth.reponse.identity_provider="https://accounts.google.com/"
+#auth.login_text="Login with Google"
+
+# MIT OAuth Example Config
+auth.client.token_url="https://oidc.mit.edu/token"
+auth.client.profile_url="https://oidc.mit.edu/userinfo"
+auth.response.name_field="name"
+auth.response.sub_field="sub"
+auth.response.email_field="email"
+auth.reponse.identity_provider="https://oidc.mit.edu/"
```

Comment:
Oh wait, your comment made me realize that all of these values really need to be ENV overridable. Otherwise, someone needs to maintain this file out of version control or on their own fork or they won't be able to push to something like Heroku. Maybe it's worth breaking this out to it's own config file while adding in that mess of additional optional ENV overrides.